nemo/lightning/io/connector.py

```diff
@@ -275,6 +276,8 @@ def nemo_load(
         else:
             _trainer.strategy.load_checkpoint(path)
 
+        model = model.to(torch.bfloat16)
```

Comment:
What would be a more elegant way of doing this instead of hard-coding to bf16?

Reply:
@akoumpa don't we have a mechanism for this for our imports?